testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/console/authentication/OTPPolicyTest.java

```diff
@@ -66,42 +67,58 @@ public void otpPolicyTest() {
     }      
     
     @Test
+    @Ignore //KEYCLOAK-2051 when you close notification, it is not displayed again
```

Comment:
The test is passing on Travis right now but it doesn't catch the bug. So it has to be fixed again. And the fix brings KEYCLOAK-2051.